main.py

```diff
@@ -0,0 +1,40 @@
+import argparse
+import tensorflow as tf
+from model.util.networkconfig import yamlconfig as networkconfig
+
+def main():
+    """ A main method that creates the model and starts training it """
+    # Parse arguments
+    parser = argparse.ArgumentParser(add_help=True)
+    parser.add_argument('configs', metavar='C', type=int, nargs='+',
+                        help='Config number to use (can be multiple)')
+    args = parser.parse_args()
+
+    for conf in args.configs:
+        config_file = networkconfig[conf]
+        with tf.Session() as sess:
+            network_model = get_model(config_file['type'], config_file, sess)
+            network_model.train()
+            network_model.close_writers()
+        tf.reset_default_graph()
+
+def get_model(model_type, config_file, session):
```

Comment:
Lägg in all models i en dict istället så slipper man massa elif, och jömförelser